app/models/dms.rb

```diff
@@ -14,7 +14,7 @@ def exists?
     return false if @dms.blank?
     res = request(:start_search)
     request :close_search
-    res.code.to_i == 200
+    Nokogiri::XML(res.body).root.text.to_i == 1
```

Comment:
```suggestion
    res.code.to_i == 200 && Nokogiri::XML(res.body).root.text.to_i == 1
```
Der Body des Response sollte nur bei Antworten mit Status OK geparst und verglichen werden.